packages/types/src/jwtv2.ts

```diff
@@ -17,6 +17,7 @@ export interface JwtHeader {
   alg: string;
   typ?: string;
   cty?: string;
+  // eslint-disable-next-line
```

Comment:
❓ Mind explicitly defining what we're disabling here? Thanks!

Reply:
the line below no longer errors, so i'm removing this line